Tests/FluentAssertions.Specs/FluentAssertions.Specs.csproj

```diff
@@ -29,23 +29,21 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net47'">
-    <PackageReference Include="System.Collections.Immutable" Version="1.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0'">
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="[16.2.0]" />
```

Comment:
Why removing this?

Reply:
Apparently it's no longer needed. It was removed automatically by the IDE.

Reply:
Ahh, it's because an older version comes with `xunit.runner.visualstudio`
![image](https://user-images.githubusercontent.com/919634/104108389-8c8c3a80-52c4-11eb-82a5-0ac61424c96b.png)
I can't tell if it makes a difference for us, there's a gazillion changes between 15.0.0 and 16.2.0
https://github.com/Microsoft/vstest-docs/blob/master/docs/releases.md

Reply:
I tried checking out the branch
* the `Microsoft.NET.Test.Sdk` dlls are not included in the netcoreapp2.0 bin folder.
* I could run the tests from VS
* I could run the test from CLI

So I'm fine with deleting it.